tests/ResponseTest.php

```diff
@@ -108,4 +108,19 @@ public function testMapping(): void
         $response = new JSendResponse(Status::translate(false, [false => StatusInterface::STATUS_ERROR]), null);
         $this->assertTrue($response->getStatus()->isError());
     }
+
+    public function testPsr7Response()
+    {
+        $response = JSendResponse::success(['Erfolgreich!'])->asResponse(200);
+        $this->assertEquals(200, $response->getStatusCode());
+        $this->assertEquals('{"status":"success","data":["Erfolgreich!"]}', $response->getBody()->getContents());
+
+        $response = JSendResponse::fail(['Irgendwas lief schief'])->asResponse(400);
+        $this->assertEquals(400, $response->getStatusCode());
+        $this->assertEquals('{"status":"fail","data":["Irgendwas lief schief"]}', $response->getBody()->getContents());
+
+        $response = JSendResponse::error('Es ist ein Fehler aufgetreten', 404)->asResponse(500);
```

Comment:
Das ist irgendwie komisch. Wieso wird er Code doppelt definiert?

Reply:
JSend Error hat ein optionales Feld das Code heißt. Siehe https://labs.omniti.com/labs/jsend. Davon unabhängig (weil ja auch optional) gibt man den Status-Code des Response an.

Reply:
Man könnte die Angabe des Response Status-Codes auch optional lassen und in dem Fall den Code von JSend übernehmen, wenn du das möchtest.